locust/contrib/fasthttp.py

```diff
@@ -467,8 +467,8 @@ def text(self) -> Optional[str]:
                 # No information, try to detect
                 if not self.encoding:
                     self.encoding = str(detect(self.content)["encoding"])
-
-        return str(self.content, self.encoding, errors="replace")
+        if self.encoding != "None":
```

Comment:
are you intentionally comparing with the _string_ None, or can this be replaced with `if not self.encoding is None:`?

Reply:
Since encoding is a string and in the previous line we have assinged, encoding = str(detect()) it is not actually Null or None it is string `None`

Reply:
Ah. Then that should be changed.

Reply:
Done